src/api/controllers/WalletController.js

```diff
@@ -1,14 +1,17 @@
 const PointSDKController = require('./PointSDKController');
 const ethereumjs = require('ethereumjs-util');
-const blockchain = require('../../network/providers/ethereum');
-const {getNetworkPublicKey} = require('../../wallet/keystore');
+const ethereum = require('../../network/providers/ethereum');
```

Comment:
I haven't found a place where we check the WalletController methods are called by a privileged host `point`.